recipe/meta.yaml

```diff
@@ -19,12 +19,12 @@ requirements:
     - pip
   run:
     - python >=3.5
+    # This is a circular dependency b/c sphinx depends on this package now :-/
+    - sphinx
```

Comment:
As a run dependency it should not be a problem. I guess that the time I packaged this conda-build had a bur or something.